CONTRIBUTING.md

```diff
@@ -6,21 +6,22 @@ This guide outlines the process of landing patches in gocql and the general appr
 
 ## Background
 
-The goal of the gocql project is to provide a stable and robust CQL driver for Go. gocql is a community driven project that is coordinated by a small team of core developers.
+The goal of the gocql project is to provide a stable and robust CQL driver for Go.  This is a community driven project that is coordinated by a small team of developers in and around the Apache Cassandra project.  For security, governance and administration issues please refer to the Cassandra Project Management Committee.
 
 ## Minimum Requirement Checklist
 
 The following is a check list of requirements that need to be satisfied in order for us to merge your patch:
 
-* You should raise a pull request to gocql/gocql on Github
+* You should raise a pull request to apache/cassandra-gocql-driver on Github
 * The pull request has a title that clearly summarizes the purpose of the patch
 * The motivation behind the patch is clearly defined in the pull request summary
-* Your name and email have been added to the `AUTHORS` file (for copyright purposes)
+* You agree that your contribution is donated to the Apache Software Foundation (appropriate copyright is on all new files)
 * The patch will merge cleanly
 * The test coverage does not fall below the critical threshold (currently 64%) 
```

Comment:
Current coverage seems to be 50.4% of statements :disappointed: 
We should probably update the number here and open an issue to display the coverage in pull requests as a comment (perhaps even enforce the coverage rule in a CI check).